mod.sp

```diff
@@ -7,6 +7,20 @@ locals {
   }
 }
 
+variable "tag_dimensions" {
+  type        = list(string)
+  description = "A list of tags to add as dimensions to each control."
```

Comment:
Maybe the description should state the caveats:
- The added dimension column must appear in _every_ control query ( i assume we are generalizing this?)
- The added dimension column must be _unique_  in every control query (when joining tables)